x/multisig/keeper/grpc_query.go

```diff
@@ -67,8 +67,9 @@ func (q Querier) Key(c context.Context, req *types.KeyRequest) (*types.KeyRespon
 
 	participants := slices.Map(key.GetParticipants(), func(p sdk.ValAddress) types.KeyResponse_Participant {
 		return types.KeyResponse_Participant{
-			Weight: key.GetWeight(p),
-			PubKey: funcs.MustOk(key.GetPubKey(p)).String(),
+			Address: p.String(),
+			Weight:  key.GetWeight(p),
+			PubKey:  fmt.Sprintf("0x%s", funcs.MustOk(key.GetPubKey(p)).String()),
```

Comment:
I'm thinking of making `pubkey.String()` prefix the `0x` everytime for consistency.

Reply:
I don't like adding that `0x`. `0x` is a very ethereum decision instead of crypto one. It's also not parsable by the golang crypto lib we use.